pytest/tests/sanity/recompress_storage.py

```diff
@@ -80,7 +93,7 @@ def do_test_recompress_storage(self, *, archive: bool) -> None:
         time.sleep(5)
         sanity.rpc_tx_status.test_tx_status(self.nodes, nonce_offset=3)
 
-    def _call(self, node: cluster.LocalNode,
+    def _call(self, node: cluster.LocalNode, prefix: str,
```

Comment:
with the prefix we'll have three separate output files: 
- stderr (this is from the neard run and it doesn't go through this method hence no prefix)
- recompress-storage-stderr stderr
- view-state-stderr